README.md

```diff
@@ -12,8 +15,30 @@ Once integrated, your marketing team will be able to better understand your app
 send them timely push notifications via FCM.
 
 ## Installation
-
-[//]: # (TODO publish the SDK and document install steps)
+1. Include the [JitPack](https://jitpack.io/#klaviyo/klaviyo-android-sdk) repository in your project's build file
+    ```groovy
+    allprojects {
+        repositories {
+            // build.gradle.kts
```

Comment:
nit: Maaaaaybe it'll be a little clearer to call out in the comment specifically something like `Use this in your build.gradle.kts if you are using Kotlin Gradle`

I just totally foresee someone coming in and copy/pastig this block as-is into their build script.

Reply:
Or maybe even separate the two step instructions entirely based on what build.gradle they may be using